package.json

```diff
@@ -10,11 +10,17 @@
     "test": "jest --watchAll"
   },
   "dependencies": {
+    "axios": "^1.7.2",
+    "cloudinary": "^2.2.0",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
+    "formidable": "^3.5.1",
+    "i": "^0.3.7",
```

Comment:
this looks like it happened by accident. To remove this dependency, you can run
`npm uninstall i` in your terminal